### PR TITLE
Solve hono-influxdb-connector issues 57 60 61

### LINKED ISSUE #57: add deployment files for the Hono-InfluxDB-Connector
The deployment of the Eclipse Kuksa Cloud should contain the required files and scripts for the deployment of the Hono-InfluxDB-Connector as part of the Eclipse Kuksa Cloud.

### PR DESCRIPTION
**Pull request by selobosi**
Fixes #57 #60 #61

### CODE REVIEW

File: utils/hono-influxdb-connector/README.md [hunk 1]
```diff
@@ -25,25 +25,58 @@ To build the Hono-InfluxDB-Connector navigate to its directory.
...
+
+## Deployment
+
+To deploy in a Kubernetes cluster, run
+
+```bash
+./deploy.sh \
+    <Cluster-internal URL of InfluxDB, typically http://influxdb.hono:8086> \
+    <Cluster-internal hostname of QPID dispatch router, typically hono-dispatch-router-ext.hono> \
+    <Cluster-internal port of QPID dispatch router, typically 5671> \
```
Comment (eriksven) [inline utils/hono-influxdb-connector/README.md]: Why do we have first a URL (for the InfluxDB) and then the combination of hostname and port (QPID dispatch router). Could we only give a URL for the dispatch router as well?

**Comment by eriksven** [comment]
@unly what do you think?

File: utils/hono-influxdb-connector/src/main/kubernetes/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Copyright Bosch Software Innovations GmbH [and others], 2019. Part of the Eclipse Kuksa Project.
+#
+# All rights reserved. This configuration file is provided to you under the
+# terms and conditions of the Eclipse Distribution License v1.0 which
+# accompanies this distribution, and is available at
+# http://www.eclipse.org/org/documents/edl-v10.php
+---
+apiVersion: extensions/v1beta1
```
Comment (unly) [inline utils/hono-influxdb-connector/src/main/kubernetes/deployment.yaml]: deployment is ga by now. `apps/v1`

File: utils/hono-influxdb-connector/src/main/kubernetes/configmap.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Copyright Bosch Software Innovations GmbH [and others], 2019. Part of the Eclipse Kuksa Project.
```
Comment (unly) [inline utils/hono-influxdb-connector/src/main/kubernetes/configmap.yaml]: I thought the deployment files go to `deploy/kubernetes/hono-influxdb-connector` not with the source code

File: utils/hono-influxdb-connector/src/main/kubernetes/configmap.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: hono-influxdb-connector-configmap
+  namespace: hono
+data: 
+  INFLUXDB_URL: ""
+  QPID_ROUTER_HOST: ""
+  QPID_ROUTER_PORT: "5671"
+  HONO_USER: ""
```
Comment (unly) [inline utils/hono-influxdb-connector/src/main/kubernetes/configmap.yaml]: put the username and password in a secret

File: utils/hono-influxdb-connector/src/main/resources/trusted-certs.pem [hunk 5]
```diff
@@ -0,0 +1,65 @@
+-----BEGIN CERTIFICATE-----
```
Comment (unly) [inline utils/hono-influxdb-connector/src/main/resources/trusted-certs.pem]: I removed them from the resources as those demo files should not go with the sources. We can track the respective hono demo certs at the deployment directories

**Pull request merged**